### DISCUSSION TITLE
Using Sysprep file while provisioning Windows VM

### DISCUSSION THREAD

**Discussion by ravitejajyosula** [oVirt/terraform-provider-ovirt#320] [Q&A]
Hello Everyone

Did anyone tried **sysprep** while creating a new VM from a template.

We have a scenario where we want to create a VM from a template and change its IP to static and hostname. We are able to achieve this using hardcoded answerfile in the drive but we want to pass dynamic values for both hostnames and IP's from terraform. 

We followed this [link ](https://access.redhat.com/documentation/en-us/red_hat_enterprise_virtualization/3.6/html/virtual_machine_management_guide/sect-using_sysprep_to_automate_the_configuration_of_virtual_machines), but it didnt help. We want to create multiple VM in parallel. 

If any one tried this, can you guide us here.

Appreciate your help 

Thanks in Advance.

**Comment by ghost** [comment]
Hey @ravitejajyosula this question may be better asked on the ovirt-users mailing list as it is not tied to the Terraform provider and more people are reading that list than discussions here.
